nixos/modules/programs/starship.nix

```
@@ -0,0 +1,57 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.starship;
+
+  settingsFormat = pkgs.formats.toml { };
+
+  settingsFile = settingsFormat.generate "starship.toml" cfg.settings;
+
+in {
+  options.programs.starship = {
+    enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Enable the Starship shell prompt.
+      '';
+    };
```

```suggestion
    enable = mkEnableOption "Starship the shell prompt";
```
`Whether to enable` is automatically prefixed when using mkEnableOption and `.` is automatically added